cpp/include/cudf/table/experimental/row_operators.cuh

```diff
@@ -151,8 +214,11 @@ namespace lexicographic {
  * `aac < abb`.
  *
  * @tparam Nullate A cudf::nullate type describing whether to check for nulls.
+ * @tparam PhysicalElementComparator A relational comparator functor, defaults to `NaN` aware
+ * relational comparator that evaluates `NaN` as greater than standard values.
```

Comment:
It's unclear to me if "standard" values would include extrema like positive infinity. This makes it a broader statement. This may need to be adapted below to say `all non-null values` or something like that...
```suggestion
 * relational comparator that evaluates `NaN` as greater than all other values.
```